openstack_dashboard/static/app/core/openstack-service-api/swift.service.js

```diff
@@ -116,8 +116,8 @@
      * @returns {Object} An object with the metadata fields.
      *
      */
-    function getContainer(container, ignoreError) {
-      var promise = apiService.get(service.getContainerURL(container) + '/metadata/');
+    function getContainer(container, ignoreError, config = {}) {
+      var promise = apiService.get(service.getContainerURL(container) + '/metadata/', config);
```

Comment:
Default args in a function signature isn't standard Javascript, perhaps browsers can handle it these days, but... I think you're better off being safe and doing like:

```suggestion
      var promise = apiService.get(service.getContainerURL(container) + '/metadata/', config || {});
```